scwx-qt/source/scwx/qt/manager/marker_manager.cpp

```diff
@@ -36,8 +36,10 @@ class MarkerManager::Impl
    explicit Impl(MarkerManager* self) : self_ {self} {}
    ~Impl() { threadPool_.join(); }
 
-   std::string                                markerSettingsPath_ {};
+   std::string                                markerSettingsPath_ {""};
    std::vector<std::shared_ptr<MarkerRecord>> markerRecords_ {};
+   std::unordered_map<types::MarkerId, size_t> idToIndex_ {};
```

Comment:
Minor: Could combine the vector and unordered map to simplify (`std::unordered_map<types::MarkerId, std::shared_ptr<MarkerRecord>>`)

Reply:
That could work, but I indented for order to be persistent. (In the future [probably part 3], I plan on adding key binds to center the map on the next/previous marker). I could switch to using `std::unordered_map<types::MarkerId, std::shared_ptr<MarkerRecord>>` instead of `idToIndex_` which would simplify, but duplicate the `shared_ptr`.